rakelib/test.rake

```diff
@@ -16,6 +16,11 @@ namespace "test" do
     RSpec::Core::Runner.run(["--fail-fast", *Rake::FileList["spec/**/*.rb"]])
   end
 
+  task "plugins" => [ "bootstrap","plugin:install-all" ] do
+    pattern = "vendor/bundle/jruby/1.9/gems/logstash-*/spec/{input,filter,codec,output}s/*_spec.rb"
+    sh "./bin/logstash rspec #{pattern}"
```

Comment:
This call require us to be at the root of logstash, we should use the `Logstash::LOGSTASH_HOME` constant to generate the path so we can run this task anywhere.